src/routers/mailingList/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { MailingListRequestBody } from './types';
+
+export const isMailingListRequestBody = (
```

Comment:
I think writing one of these for each route will be way too tedious😅

Reply:
Yeah 😅 I had done a little bit of looking and found this fairly popular package [express-validator](https://github.com/express-validator/express-validator). I decided not to include it  for this route since it's probably something we should consider and agree on. Lemme know what you think of it when you get a chance.